src/main/java/com/rtb/projectmanagementtool/user/AuthOps.java

```diff
@@ -0,0 +1,46 @@
+package com.rtb.projectmanagementtool.user;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.api.datastore.Query.*;
+import java.util.*;
+
+public final class AuthOps {
+
+  private DatastoreService datastore;
+  public String cookieName;
+  public String[] cookieValues;
+  public Cookie userCookie;
+
+  public AuthOps(DatastoreService datastore) {
+    this.datastore = datastore;
+    cookieName = "loginStatus";
```

Comment:
We're not just storing a single status here, since we said we'd also use this to track who is logged in.  So maybe "loggedInUserId" or "sessionUserId" would work instead